### ISSUE TITLE
Outdated template version message should be configurable

### ISSUE THREAD

**Issue by samkeet** [coder/coder#12852]
Currently when a workspace is outdated, and a user SSH's into it they are displayed with the text 
```shell
👋 Your workspace is outdated! Update it here: {coder url}
```

We should allow this message to be made configurable and allow skipping exposing the coder URL. As a administrator team, we would want to not bother users with details of the coder url and be concerned with the updates

**Comment by kylecarbs** [comment]
Agreed. We've gotten numerous requests to make our notification system customizable. cc @stirby
